virttest/utils_test/__init__.py

```diff
@@ -1193,7 +1224,8 @@ def run_virt_sub_test(test, params, env, sub_type=None, tag=None):
             subtest_dirs += data_dir.SubdirList(generic_subdir,
                                                 bootstrap.test_filter)
 
-        for specific_subdir in asset.get_test_provider_subdirs(params.get("vm_type")):
+        for specific_subdir in asset.get_test_provider_subdirs(
+                params.get("vm_type")):
```

Comment:
And two style-fixes... I's better to separate those, but don't take it as an obligation now.